### DISCUSSION TITLE
About limitation or possible risk

### DISCUSSION THREAD

**Discussion by dungnguyennvd** [Budibase/budibase#9737] [General]
I’m consider to build a new logistics solution use Golang. And use this project, so, i want to ask expert in this project about limitation. Some of those:
1. Performance when user is more than 100 user 
2. Possible to scale?, when it limit of user or record?
Thanks very much

**Comment by melohagan** [comment]
Hey @dunggnguyen 

I'm not sure that this question has anything specifically to do with Budibase?

Just out of curiosity I fired your question into ChatGPT and this was the answer:

> As a language model, I can provide information about the capabilities and limitations of Golang, but it's important to note that the performance and scalability of your logistics solution will also depend on other factors such as the design of your application, your infrastructure, and your data storage.

> That being said, Golang is known for its high performance and efficient concurrency model, which makes it well-suited for building high-performance applications. It's designed to handle large-scale, multi-user systems and can handle hundreds or even thousands of users without significant performance degradation.

> Regarding scaling, Golang has built-in support for concurrency, which allows you to take advantage of multiple cores and distribute workloads across multiple threads. This can help you scale your application by increasing the number of concurrent requests it can handle.

> In addition, Golang has a robust ecosystem of third-party libraries and frameworks that can help you scale your application. For example, Kubernetes is a popular container orchestration platform that can be used to manage and scale Golang applications.

> However, it's important to keep in mind that scalability is not just a matter of technology, but also of architecture and design. You'll need to carefully consider how your application is architected and how it handles data storage and retrieval in order to ensure that it can scale effectively.

> In summary, Golang is well-suited for building high-performance, scalable applications, and it should be able to handle hundreds or even thousands of users without significant performance degradation. However, achieving scalability requires careful consideration of your application's architecture and design, as well as the infrastructure and data storage you use.
